doc/content/concepts/devicetemplates/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Device Templates"
+description: ""
+weight: 1
+---
+
+Device templates can be used to quickly create large amounts of end devices with common settings. Templates can be created from existing devices or converted from an input file. You can create end devices directly from templates. See [Creating Templates](./creating), [Converting Templates](./converting) and [Executing Templates](./executing).
+
+Templates can also be be mapped with other templates to combine fields. For example, you can convert root key provisioning data to device templates and map that with a template containing the device profile from a device repository. See [Mapping Templates](./mapping).
+
+For device makers, the tooling supports assigning the LoRaWAN `JoinEUI` and `DevEUI` from a range automatically to create a template file that can be used in mapping or executing devices. See [Assigning EUIs](./assigning-euis).
+
+To work with device templates, use the command-line interface (CLI). See [Getting started with the command-line interface](../../gettingstarted/cli).
```

Comment:
> Getting started with the command-line interface

We should think about guidelines for using title case

Reply:
Removed capital G. This is not a page title